README.event-manager.md

```diff
@@ -103,7 +103,9 @@ may also be an empty string.
 The command is an uzbl command which may use format string replacement:
 
 * `%s`: The string as given.
-* `%r`: The string, escaped and quoted for uzbl.
+* `%r`: The string, escaped for uzbl.
+  This format used to also wrap the string in single-quotes. The same result
+  can still be achived by include the quotes in the format string (e.g `'%r'`)
```

Comment:
yep. that looks to be it. `include` would also be trivial to change to use split.

I think either way it's not a big worry for porting as it's not a fundamental change but just some escaping noise. I'll can take a look and see how much would break from changing uri.